### DISCUSSION TITLE
Where do i download the program executables ?

### DISCUSSION THREAD

**Discussion by nononymousse** [PCBox/PCBox#55] [Q&A]
Hi ^^
your project look really interesting but i can't see on the readme fo the Github or on your website or to download a executable(s). Is the souce code only avaliable and we have to build it ouselves ?

**Comment by Torinde** [comment]
See #33

**Comment by nononymousse** [comment]
This answer is only for windows, but even that with one should i get ? and Actions are not made to build releases of the binary, it's basically to test compilation

and on top of that, why not saying it on the readme of the project ? Like it's a basic thing to be able to download the software

**Reply by Torinde** [reply]
I'm also interested in the differences between the various artifacts, but I don't know. That and your other questions can be answered by the developer.

**Reply by bachimusprime** [reply]
I do say it in the readme. It says right there that binaries are under the Actions tab. Anyway, go to the CMake workflow and pick up the Qt NDR MINGW32 builds

**Reply by nononymousse** [reply]
You say to go there but not what to download for each OS, thanks for letting me know to take the Qt NDR MINGW32 build but why this one ? what are the other ones do ? woudn't be more simpler to put releases on the releases page ?

**Reply by bachimusprime** [reply]
I would rather not, given that I'd like everyone to be on the latest code when using PCBox, and if I did versioned releases, people would invariably stick with the "stable" releases instead of using the latest code.

**Reply by nononymousse** [reply]
even if it's unstable ? that dosen't make sense to me
what about posting them there anyway ? this will be **way** simpler that having to dig on issues to understand how to download the software

**Reply by Torinde** [reply]
I assume qeeg put "stable" in quotes, because releases are equally stable, regardless if versioned or not, e.g. there is no team 10 QA engineers playing Doom multiplayer to test stability before releasing...

@qeeg, what about documenting somewhere what's the difference between the artifacts, e.g. why using Qt, why NDR, why MINGW32?

**Reply by nononymousse** [reply]
yes, that's a good idea ^^
(btw sorry if i was mean on y answers)

**Reply by KoghaandSooga** [reply]
There's no 'Roms' in the rom releases for this.

**Reply by bachimusprime** [reply]
Yes there are, stop being fucking obtuse.

**Comment by KoghaandSooga** [comment]
> Hi ^^ your project look really interesting but i can't see on the readme fo the Github or on your website or to download a executable(s). Is the souce code only avaliable and we have to build it ouselves ?

Same here. I wish for an EXE of this program. It's a shame we have to compile this as that is way beyond me and most people who might accidently stumble upon this. This feels like walking to a non compliant ADA building when all they need is a 'short' ramp over a few steps. 

Now I can understand if it's a long ramp and a lot of steps where it may wreck a lot of the architecture of said building ruining the historical value so I wonder if that's the same deal with this emulator where by releasing an exe it might break some core value?

**Reply by bachimusprime** [reply]
You don't have to compile it. Binaries are available in the Actions tab, as has been stated MANY TIMES BEFORE. Please actually read this thread before making assumptions.
